tests/backup/backup_helper.go

```diff
@@ -1322,6 +1696,15 @@ func restoreSuccessWithReplacePolicy(restoreName string, orgID string, retryDura
 	return err
 }
 
+// ValidateRestore returns a clone of the contexts (with backup objects) *after* converting the contexts to point to restored objects (and after validating those objects)
+func ValidateRestore(ctx context.Context, restoreName string, orgID string, backedupAppContexts []*scheduler.Context, namespaceMapping map[string]string, storageClassMapping map[string]string, restoreClusterConfigPath string) ([]*scheduler.Context, error) {
```

Comment:
please see if you minimise the args

Reply:
No, all those are needed. But this function will be handled in next PR anyway. we can resolve issues then...this is placeholder for demo